test/markup/powershell/flags.expect.txt

```diff
@@ -0,0 +1,4 @@
+<span class="hljs-string">&quot;Good Dog&quot;</span> <span class="hljs-literal">--match</span><span class="hljs-literal">-lower</span><span class="hljs-literal">-case</span> <span class="hljs-string">&quot;Dog&quot;</span>
+<span class="hljs-number">2</span> <span class="hljs-literal">--equality</span><span class="hljs-literal">-check</span> <span class="hljs-number">2</span>
+<span class="hljs-string">&quot;abc&quot;</span> <span class="hljs-literal">--format</span><span class="hljs-literal">-drive</span><span class="hljs-literal">-fully</span> <span class="hljs-string">&quot;def&quot;</span>
```

Comment:
So that doesn't look quite right (even if visually it's ok), perhaps the regex needs to allow `-` inside of flags not just \w and \d... the flag is "--format-drive-fully`, not 3 separate flags.

Reply:
Can you suggest a better regex? I am a little confused here @joshgoebel

Reply:
For starters I think this might work:

`/(-){1,2}[\w\d-]+/`

Or a word followed by additional -word segments?

`/(-){1,2}[\w\d]+(-[\w\d])*/`